extensions/pkg/controller/backupbucket/controller.go

```diff
@@ -42,16 +42,17 @@ type AddArgs struct {
 	// The options.Reconciler is always overridden with a reconciler created from the
 	// given actuator.
 	ControllerOptions controller.Options
-	// Predicates are the predicates to use.
-	// If unset, GenerationChangedPredicate will be used.
-	Predicates []predicate.Predicate
 	// Type is the type of the resource considered for reconciliation.
 	Type string
+	// IgnoreOperationAnnotation specifies whether to ignore the operation annotation or not.
+	// If the annotation is not ignored, the extension controller will only reconcile
+	// with a present operation annotation typically set during a reconcile (e.g in the maintenance time) by the Gardenlet
+	IgnoreOperationAnnotation bool
 }
 
 // DefaultPredicates returns the default predicates for a controlplane reconciler.
-func DefaultPredicates(ignoreOperationAnnotation bool) []predicate.Predicate {
-	if ignoreOperationAnnotation {
+func (a AddArgs) DefaultPredicates() []predicate.Predicate {
```

Comment:
Why is this now a function of the `AddArgs`? Can't we keep that the caller has to provide the "ignore operation annotation" via argument?